packages/api/lambdas/replay-sqs-messages.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const pMap = require('p-map');
+
+const { envUtils } = require('@cumulus/common');
+const { getJsonS3Object, listS3ObjectsV2 } = require('@cumulus/aws-client/S3');
+const { getQueueUrlByName, sendSQSMessage } = require('@cumulus/aws-client/SQS');
+const { getS3PrefixForArchivedMessage } = require('@cumulus/ingest/sqs');
+const Logger = require('@cumulus/logger');
+
+const logger = new Logger({ sender: '@cumulus/replay-sqs-messages' });
+
+// Get messages from S3 using queueName
+const getArchivedMessagesFromQueue = async (queueName) => {
+  let continuationToken;
+  const validMessages = [];
+  const bucket = envUtils.getRequiredEnvVar('system_bucket', process.env);
+  const stackName = envUtils.getRequiredEnvVar('stackName', process.env);
+  const prefix = getS3PrefixForArchivedMessage(stackName, queueName);
+  const params = {
+    Bucket: bucket,
+    Prefix: prefix,
+    ContinuationToken: continuationToken,
```

Comment:
I don't think you need this parameter anymore

Reply:
[`030ed97` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/030ed97f35a8892454a2f46e229a87b2bb447203#diff-50b67b19453ae7cd068a961f63d9775c6e52caadcad45e3cc0f33552900d8e3dL15)